Source/DataTypes/SvgOrient.cs

```diff
@@ -11,11 +11,13 @@ namespace Svg
 	public class SvgOrient
     {
         private bool _isAuto = true;
+        private bool _isAutoStartReverse = false;
         private float _angle;
   
 		public SvgOrient()
 		{
 			IsAuto = false;
+            IsAutoStartReverse = false;
             Angle = 0;
```

Comment:
Hm, I was sure I had already commented here - the mentiuoned formatting issue is the space/tab mix here (which has been there before, so you didn't introduce it), so it would be nice if you could just replace the tabs with spaces in this file.

Reply:
Thank you for your feedback and for accepting my contribution :)
I will fix this little formatting issue. Svg/png images will also be added to the test suite.
And I will complete the implementation to support both rendering and loading correctly this new value (my first proposal only take into account my use case for generating svg image).